src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -21,8 +18,7 @@
  * Class representing the parser for the view command.
  */
 public class ViewCommandParser implements Parser<ViewCommand> {
-
-    private static final Supplier<Boolean> ALWAYS_TRUE = () -> true;
+    private static final ParseException NO_FIELD_EXCEPTION = new ParseException(ViewCommand.HELP_MESSAGE);
```

Comment:
Do you think we need to change other ParseExceptions to static final to follow this format?